moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js

```diff
@@ -177,20 +177,26 @@ export const ProjectsListViewQueryConf = {
       filter: value => new Date(value).toLocaleDateString("en-US"),
       type: "date_iso",
     },
-    // the following column configurations only work with material table and do not have the 
+    // the following column configurations only work with material table and do not have the
     // necessary details to work with Grid Table
     task_order: {
-      type: "string",
+      type: "Array",
```

Comment:
good question -- I capitalized it because I was thinking that's the official type declaration, but `String` is also capitalized officially. we don't use these types in the config (the types are used in GridTable), but I should keep the formatting consistent.